sound/soc/sof/Makefile

```diff
@@ -1,7 +1,8 @@
 # SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
-		control.o trace.o utils.o sof-audio.o stream-ipc.o ipc4.o
+		control.o trace.o utils.o sof-audio.o stream-ipc.o ipc4.o \
```

Comment:
Can we change the commit message to something like: 
"ASoC: SOF: topology: share common token parsing code

Move the token definitions along with the code for parsing them to a common file so they can be shared with IPCv4 topology parser"